src/Dropdown.jsx

```diff
@@ -1,3 +1,6 @@
+/* eslint react/no-find-dom-node: 0 */
+// https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-find-dom-node.md
```

Comment:
Didn't update this just yet to use `ref={node => this.node = node}` because `node` for some reason is null sometimes. Specifically on dropdown toggle click. https://github.com/reactstrap/reactstrap/blob/0c1b5e48c208ac4542357534defa7681e467fbc9/src/Dropdown.jsx#L183